ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -57,6 +59,19 @@ class BaseLine extends React.Component {
             label={label}
             value={this.state.value}
             onChange={value => {
+              if (value.includes('MULTI') && value.trim().endsWith(')))')) {
```

Comment:
could you do like `onChange={value.trim() => {`
I don't think you'll ever need to preserve the extra space on the ends, and it would stop you from having to call .trim() everywhere

Reply:
or even `value={this.state.value.trim()}`